vendor/engines/projects/lib/projects/engine.rb

```diff
@@ -2,8 +2,13 @@ module Projects
 
   class Engine < Rails::Engine
 
+    def self.enable!
```

Comment:
`enable!` is never actually getting called. If I log in as a non-admin, I don't see the menu item. Admins see it because of `can :manage, :all`